keyboards/keycapsss/plaid_pad/config.h

```diff
@@ -18,8 +18,10 @@
 #define MATRIX_COL_PINS { B0, D7, D6, D5 }
 #define UNUSED_PINS     { B3, B4, B5, D4}
 
-#define ENCODERS_PAD_A { D0, B1 }
-#define ENCODERS_PAD_B { D1, B2 }
+#define ENCODERS_PAD_A { D1, B2 }
+#define ENCODERS_PAD_B { D0, B1 }
```

Comment:
This causes the encoders on my board (Bourns EC11) to become flipped.

Reply:
Hi,
this related to the brands of the rotary encoder.
I sell TT Electronics/BI rotary encoder in my shop and with my code changes it works like expected.
I tested it with Bourns Inc. encoder and you are right it's flipped.

A simple solution would be to add the following code to the `config.h`
```
// uncomment the following line to flip the encoder directions
// #define ENCODER_DIRECTION_FLIP
```

Reply:
Seems like it depends which part number it is, for the TT encoders:
![image](https://user-images.githubusercontent.com/4781841/88046872-0c4a0f80-cb94-11ea-91ab-0b62edd067e3.png)
Whereas Bourns evidently only use "A leads B":
![image](https://user-images.githubusercontent.com/4781841/88046948-297ede00-cb94-11ea-8215-52e611407ab4.png)

If the TT encoders are the ones that typically come with the board, then the issue is on my end, I'll have to suck it up and add the flip define :)